ui.frontend/src/constants.js

```diff
@@ -83,6 +83,129 @@ export const Constants = {
      */
     FORM_LANGUAGE_INITIALIZED: "AF_LanguageInitialised",
 
+    /**
+     * Event triggered when focus is changed on a form element
+     * @event module:FormView~Constants#ELEMENT_FOCUS_CHANGED
```

Comment:
The documentation should be there on the event itself in GuideBridge for easy discoverability.

Reply:
What do you mean by discoverability ? Can you explain. All the events are triggered from view layer and we don't document the view layer code today (only ui.frontend is documented). 

IMO, only thing missing in the documentation is the reference from where the event is thrown (ie) @fires annotation, an example can be seen here, https://opensource.adobe.com/aem-core-forms-components/module-FormView-Utils.html#.setupFormContainer__anchor